Source/WebKit/UIProcess/Cocoa/SOAuthorization/SOAuthorizationCoordinator.mm

```diff
@@ -46,13 +46,17 @@
 
 namespace WebKit {
 
-SOAuthorizationCoordinator::SOAuthorizationCoordinator()
+SOAuthorizationCoordinator::SOAuthorizationCoordinator(bool enabled)
 {
+    if (!enabled)
```

Comment:
Seems like we should just have not constructed it in this case, right? But I guess this is how the existing disablement works, so we know we have `m_soAuthorization` checks in all the right places...